### PR TITLE
Add empty state for individual classrooms page tabs

### PR DESCRIPTION
**Pull request by jfdoming**
## Notion ticket link

[Add empty state for archived classrooms](https://www.notion.so/uwblueprintexecs/Add-empty-state-for-archived-classrooms-793f75308ae748daafb56d5471a967ad)

## Implementation description
Adds empty states for individual tabs on the classrooms page, in case one tab is empty and the other is not. I added a new empty section for archived classrooms and reused the existing empty classrooms state for active classrooms.

User with no archived classes but at least one active:
image

User with no active classes but at least one archived:
image

User with no classes at all (existing page behaviour):
image

## Steps to test
1. Make sure you have at least one archived and one active classroom.
2. Delete all your active classrooms and view the empty state for that tab.
3. Delete all your archived classrooms and view the overall empty state.
4. Add an active classroom and view the archived tab empty state.

## What should reviewers focus on?
* 

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/common/info/messages/EmptyArchivedClassroomsMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import DisplayAssessmentsIllustration from "../../../../assets/illustrations/display-assessments.svg";
+import MessageContainer from "../MessageContainer";
+
+const EmptyArchivedClassroomsMessage = (): ReactElement => {
+  const styles = useStyleConfig("Center", { variant: "emptyMessage" });
+  return (
+    <Center __css={styles} height="100%" mt={4} pb={14}>
+      <MessageContainer
+        image={DisplayAssessmentsIllustration}
+        paragraphs={["Use the archive option on a classroom to view it here"]}
```
Comment (carissa-tang) [inline frontend/src/components/common/info/messages/EmptyArchivedClassroomsMessage.tsx:13]: i feel like this is unnecessary imo?

File: frontend/src/components/common/info/messages/EmptyArchivedClassroomsMessage.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React, { type ReactElement } from "react";
+import { Center, useStyleConfig } from "@chakra-ui/react";
+
+import DisplayAssessmentsIllustration from "../../../../assets/illustrations/display-assessments.svg";
+import MessageContainer from "../MessageContainer";
+
+const EmptyArchivedClassroomsMessage = (): ReactElement => {
+  const styles = useStyleConfig("Center", { variant: "emptyMessage" });
+  return (
+    <Center __css={styles} height="100%" mt={4} pb={14}>
```
Comment (carissa-tang) [inline frontend/src/components/common/info/messages/EmptyArchivedClassroomsMessage.tsx]: might have some conflicts from my pr

Comment (carissa-tang) [reply frontend/src/components/common/info/messages/EmptyArchivedClassroomsMessage.tsx]: fixed, but you can double check

**Review by carissa-tang — APPROVED** [review]

File: frontend/src/components/common/info/messages/EmptyArchivedClassroomsMessage.tsx (hunk 1 above)
Comment (jfdoming) [reply frontend/src/components/common/info/messages/EmptyArchivedClassroomsMessage.tsx:13]: I think it's better to be extra explicit here!

**Pull request merged**